### DISCUSSION TITLE
session.add(instance) doesn't work and neither raise any errors?

### DISCUSSION THREAD

**Discussion by subodh7591** [sqlalchemy/sqlalchemy#7218] [Usage Questions]
I created new instance of a model and try to add to the database but session.add() doesn't seem to work. It doesn't raise any exception. My python version 3.9.6

**Comment by CaselIT** [comment]
Hi,

Please look at the tutorial https://docs.sqlalchemy.org/en/14/tutorial/index.html, in particular https://docs.sqlalchemy.org/en/14/tutorial/orm_data_manipulation.html
